.github/workflows/staging-tests.yml

```diff
@@ -37,3 +42,32 @@ jobs:
           ./staging-env/bin/python -m sigstore verify --staging \
             --cert-oidc-issuer https://token.actions.githubusercontent.com \
             README.md
+
+      - name: generate an issue if staging tests fail
+        if: failure()
+        run: |
+          cat <<- EOF > /tmp/staging-instance-issue.md
+          ## Staging instance failure
+
+          A scheduled test against Sigstore's staging instance has failed.
+
+          This suggests one of three conditions:
+
+          * A backwards-incompatible change in a Sigstore component;
+          * A regression in \`sigstore python\`;
+          * A transient error.
+
+          The full CI failure can be found here:
+
+          $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
+          EOF
+
+      - name: open an issue if the staging tests fail
+        if: failure()
+        uses: peter-evans/create-issue-from-file@v4
+        with:
+          title: "[CI] Integration failure: staging instance"
+          # created in the previous step
+          content-filepath: /tmp/staging-instance-issue.md
+          labels: bug,component:cicd,component:tests
+          assignees: woodruffw,di,tetsuo-cpp
```

Comment:
Flagging: I put the three of us as assignees, but maybe someone from Fulcio or Rekor also makes sense?

Reply:
I think this is fine for us to triage.